qa/tasks/cephfs/test_volumes.py

```diff
@@ -414,6 +453,44 @@ def test_default_uid_gid_subvolume_group(self):
         # remove group
         self._fs_cmd("subvolumegroup", "rm", self.volname, group)
 
+    def test_subvolume_group_ls(self):
+        # tests the 'fs subvolumegroup ls' command
+
+        subvolumegroups = []
+
+        #create subvolumegroup1 and make sure it exists
```

Comment:
No need to repeat the commands three times. Do this in a loop as mentioned above.